target-riscv/cpu.c

```diff
@@ -96,16 +92,7 @@ static void riscv_cpu_disas_set_info(CPUState *s, disassemble_info *info) {
 static void riscv_cpu_realizefn(DeviceState *dev, Error **errp)
 {
     CPUState *cs = CPU(dev);
-    RISCVCPU *cpu = RISCV_CPU(dev);
     RISCVCPUClass *mcc = RISCV_CPU_GET_CLASS(dev);
-    CPURISCVState *env = &cpu->env;
-
-    /* Enable GC ISA */
-    set_feature(env, RISCV_FEATURE_RVM);
```

Comment:
One other thing: the TB cache is global, but when risc-v socs start showing up on the market they're very likely to be heterogenous, and we need something in tb_flags to prevent translations for core A from being inappropriately reused on core B with different extensions.